### DISCUSSION TITLE
pgproto3 DataRow.MarshalJSON  will turn string which contains  LF or VT into HEX string

### DISCUSSION THREAD

**Discussion by DengChan** [jackc/pgx#1863] [Q&A]
I'm developing a web pg client using pgproto3, but I found that when I using DataRow.MarshalJSON to parse the DataRow message,  it will turn string which contains  LF or VT into HEX string.

https://github.com/jackc/pgx/blob/6f2ce92356f35263258ef7baa4cde86a3e1c6255/pgproto3/data_row.go#L97-L100
Because the ASCII of  LF is 9 , and the ASCII of  VT is 11.  

But LF AND VT are often used in some text column.

Could we have some special treatment for these frequently-used ASCII ?

**Comment by jackc** [comment]
I don't think this can be changed at this point in a backwards compatible manner. I think you will need to wrap or rename the type to override the JSON logic.
